src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -1396,6 +1402,29 @@ internal static bool OnServerCertificateCustomValidationCallback(object sender,
             return false;
```

Comment:
Should we also check and throw here?

Reply:
I think we better. The major issue is that if we don't, we get a pretty cryptic (pun intended) error.